### PR TITLE
EOS:16996 - Add post_install validation in 's3_setup post_install' 

### PR DESCRIPTION
**Pull request by amitwac2608**
Signed-off-by: Amit Kumar <amit.kumar@seagate.com>

### CODE REVIEW

File: scripts/provisioning/s3_setup [hunk 1]
```diff
@@ -362,6 +362,14 @@ then
         exit 1
     fi
 
+    echo "INFO: copying ssl certificate.."
+    cp -f /opt/seagate/cortx/s3/install/haproxy/ssl/s3.seagate.com.pem /etc/ssl/stx-s3/s3/
```
Comment (sachitanands) [inline scripts/provisioning/s3_setup:366]: pls check if this ssl filename is same as mentioned in "frontend s3-main" section of haproxy.cfg file.

Comment (amitwac2608) [reply scripts/provisioning/s3_setup:366]: currently we have commented out the ssl file line in the haproxy.cfg file.

@bkirunge7 Please review this if s3.seagate.com.pem is the right ssl file name and it is being copied to right location.

Comment (amitwac2608) [reply scripts/provisioning/s3_setup]: I have removed the ssl configuration section as of now, as we need more answers from Provisioner team.

**Review by AjinkyaDhumal — APPROVED** [review]

**Review by pratyush-seagate — APPROVED** [review]
LGTM, +1

**Comment by amitwac2608** [comment]
Jenkins job on latest commit **PASSED**: http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/S3server-7.8.2003/502/

**Pull request merged**